r_gl.cpp

```diff
@@ -68,7 +136,7 @@ bool GLRender::Init(void)
         SDL_WINDOWPOS_UNDEFINED,
         WINDOW_WIDTH,
         WINDOW_HEIGHT,
-        SDL_WINDOW_OPENGL | SDL_WINDOW_RESIZABLE | SDL_WINDOW_ALLOW_HIGHDPI
+        SDL_WINDOW_OPENGL | SDL_WINDOW_ALLOW_HIGHDPI
```

Comment:
Good question! Forgot to mention that. On resize the FBO's textures have to be recreated since I am going to use a newer way of texture layout for the offscreen framebuffers in the upcoming iteration that are faster than regular textures but don't allow resizing by default. This is just a precaution so that resizing will be handled properly in the future.